### PR TITLE
Reconfigure chromedriver setup to always be in sync with google chrome

### PR DESCRIPTION
**Pull request by jonathansamines**
From [chromedriver@79](https://github.com/giggio/node-chromedriver/pull/230), supported was added to keep the chromedriver version in sync with the locally installed google chrome version following the [chrome driver version policy](https://chromedriver.chromium.org/downloads/version-selection).

I am configuring that support, so that we don't have to manually keep those versions in sync which currently make the test fail everytime google chrome gets updated

### CODE REVIEW

**Comment by jonathansamines** [comment]
From `chromedriver@80` at least Node 10 is required. We cannot further upgrade that dependency, as that would break the current test matrix.

**Comment by skratchdot** [comment]
lgtm. imo we could just drop support for node 8 since it is no longer in lts

**Comment by skratchdot** [comment]
Actually, it looks like the package-lock.json changes you submitted have the registry:
`artifactory.engtools.xoom.com`- can you change those to be the public npm registry?

File: package-lock.json [hunk 1]
```diff
@@ -24,6 +24,34 @@
...
     },
+    "@testim/chrome-version": {
+      "version": "1.0.7",
+      "resolved": "https://artifactory.engtools.xoom.com:443/artifactory/api/npm/npmjs-virtual/@testim/chrome-version/-/chrome-version-1.0.7.tgz",
+      "integrity": "sha1-DNkVeF7EGQ8Io6asybYfw4+18ak=",
+      "dev": true
+    },
+    "@types/glob": {
+      "version": "7.1.2",
+      "resolved": "https://artifactory.engtools.xoom.com:443/artifactory/api/npm/npmjs-virtual/@types/glob/-/glob-7.1.2.tgz",
```
Comment (skratchdot) [inline package-lock.json:35]: change registry to public npm (for all package-lock.json changes)

Comment (jonathansamines) [reply package-lock.json]: @skratchdot I just updated these references to the public registry

**Comment by skratchdot** [comment]
> @skratchdot Should I upgrade the project configuration to remove Node 8 support (and update the chromedriver version to latest)?

updating chromedriver makes sense to me.  for the node 8 question: i think @grawk should answer that. i mean, node 8 is out of LTS, but maybe other nemo packages still have it listed in their travis files (and all should be updated at the same time)? i'd be fine with it being removed.

**Comment by grawk** [comment]
Sorry, it's been a while since I've looked at this repo.

**Pull request merged**